com.unity.cinemachine/Editor/Editors/CinemachineConfiner2DEditor.cs

```diff
@@ -33,48 +34,69 @@ public override VisualElement CreateInspectorGUI()
 
             var volumeProp = serializedObject.FindProperty(() => Target.BoundingShape2D);
             ux.Add(new PropertyField(volumeProp));
-            ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.Damping)));
-            var oversizedCameraHelp = ux.AddChild(new HelpBox(
-                "The camera window is too big for the confiner. Enable the Oversize Window option.",
-                HelpBoxMessageType.Info));
-            
-            ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.OversizeWindow)));
-            
             TrackVolume(volumeProp);
             ux.TrackPropertyValue(volumeProp, TrackVolume);
             void TrackVolume(SerializedProperty p)
             {
                 var c = p.objectReferenceValue;
-                boundsHelp.SetVisible(!(c is PolygonCollider2D || c is BoxCollider2D || c is CompositeCollider2D));
+                boundsHelp.SetVisible(!(c is null || c is PolygonCollider2D || c is BoxCollider2D || c is CompositeCollider2D));
 
                 var cc = c as CompositeCollider2D;
                 polygonsHelp.SetVisible(cc != null && cc.geometryType != CompositeCollider2D.GeometryType.Polygons);
             }
+            
+            ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.Damping)));
+            ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.SlowingDistance)));
+            
+            var oversizedCameraHelp = ux.AddChild(new HelpBox(
+                "The camera window is too big for the confiner. Enable the Oversize Window option.",
+                HelpBoxMessageType.Info));
+
+            var owner = Target.VirtualCamera;
+            if (owner != null)
+            {
+                UpdateConfinerLensCache();
+                var lensProperty = new SerializedObject(owner).FindProperty("Lens");
```

Comment:
This is a nice thought, but it's also possible to add a confiner to a non-CmCamera - e.g. a mixing camera - and that doesn't have a "Lens" property.  So this code will fail in that case.